pennylane_lightning/core/src/simulators/lightning_kokkos/gates/GateFunctorsGenerator.hpp

```diff
@@ -21,7 +21,7 @@
 /// @cond DEV
 namespace {
 using namespace Pennylane::Util;
-using Kokkos::Experimental::swap;
+using Kokkos::kokkos_swap;
```

Comment:
How far back does this run (i.e. can we still build with Kokkos 4.0)? Do we care?

Reply:
The Kokkos exp option was deprecated in 4.3.0, and replaced in 4.3.01. So, as long as we have 4.3.00 and newer I think it is fine. CF has Kokkos 4.3 so should be fine to build for there also.